lib/sidekiq/instrument/worker.rb

```diff
@@ -65,7 +65,7 @@ def send_worker_metrics
 
       WorkerMetrics.workers_in_queue.each do |key, value|
         Statter.statsd.gauge("shared.sidekiq.worker_metrics.inqueue.#{key}", value)
-        Statter.dogstatsd&.gauge("shared.sidekiq.worker_metrics.inqueue.#{key}", value)
+        Statter.dogstatsd&.gauge("shared.sidekiq.worker_metrics.inqueue", value, tags: ["worker:#{key}"])
```

Comment:
not creating a separate method like `dd_queue_tags` bc it's only used once like this